frontend/src/components/views/ProjectileView.jsx

```diff
@@ -3,13 +3,15 @@ import { convertToRenderCoordinates, modelFiles } from "../../utils/render_utils
 import { useLoader } from "@react-three/fiber";
 import { Line } from '@react-three/drei';
 
-const renderProjectile = (projectile) => {
-    const { name, offset, spawnedAt, quaternion, scale, hp} = projectile;
-    if (!hp) return null;
+const ProjectileRender = (projectile) => {
+    const { name, offset, spawnedAt, quaternion, scale, hp } = projectile;
+    if (hp <= 0) return null; // allows lasers to disappear
```

Comment:
Huh, why do lasers need to rely on hp to disappear? Do they linger in memory?

Reply:
Yep, I didn't actually check why at the time, but it looks like all projects disappear because their hp is set to zero when they reach the end of their path. I think it would be true for all projectiles, not just lasers. Popping them out of game.projectiles would be a good improvement.